### PR TITLE
"Final" updates to ch1

### PR DESCRIPTION
**Pull request by jni**
This change is [Reviewable](https://reviewable.io/reviews/elegant-scipy/elegant-scipy/297)

### CODE REVIEW

File: markdown/ch1.markdown [hunk 1]
```diff
@@ -837,13 +846,11 @@ RPKM puts together the ideas of normalising by sample and by gene.
...
 
-Working through how RPKM is derived:
-
-Let's say:
+To work through how RPKM is derived, let's define the following values:
 
-C = Number of reads mapped to a gene
-L = exon length in base-pairs for a gene
-N = Total mapped reads in the experiment
+$C$ = Number of reads mapped to a gene<br/>
```
Comment (stefanv) [inline markdown/ch1.markdown]: These may be the cause of the problem.

**Comment by stefanv** [comment]
I turned it into a list for now; should fix it, at least.

Markdown's "two spaces at end of line to indicate a newline" is really dumb, but even that doesn't work here (either htmlbook or Jupyter converter doesn't seem to understand that subtlety).

**Comment by stefanv** [comment]
Hrmmmm.... did you disable pushes to your PR?

**Comment by stefanv** [comment]
```diff
commit 3addac8b4712b1fa81b11d98b360c9f87949a29a
Author: Stefan van der Walt <stefanv@berkeley.edu>
Date:   Sat May 13 15:52:55 2017 -0700

    Small typography hack around <br/> issues

diff --git a/markdown/ch1.markdown b/markdown/ch1.markdown
index ff5f848..8fd11ed 100644
--- a/markdown/ch1.markdown
+++ b/markdown/ch1.markdown
@@ -848,9 +848,9 @@ and the gene length.
 
 To work through how RPKM is derived, let's define the following values:
 
-$C$ = Number of reads mapped to a gene<br/>
-$L$ = exon length in base-pairs for a gene<br/>
-$N$ = Total mapped reads in the experiment
+- $C$ = Number of reads mapped to a gene
+- $L$ = exon length in base-pairs for a gene
+- $N$ = Total mapped reads in the experiment
 
 First, let's calculate reads per kilobase.
```

**Comment by stefanv** [comment]
https://gist.github.com/stefanv/806347a759040b812ccce4d69bab54e4

Not sure what the easiest way is to ship a patch without pushing it :/

**Comment by jni** [comment]
I thought that math in lists doesn't work? Anyway, trying it out...

**Comment by jni** [comment]
Huh! I'm totally weirded out, because I tried just removing the `br`s yesterday and still got a build error involving the section tags. But anyway, this passed, so here it is! =)

**Comment by stefanv** [comment]
¯\_(ツ)_/¯

**Pull request merged**